src/lib/viewport-uniforms.js

```diff
@@ -23,28 +23,13 @@ function calculateMatrixAndOffset({
   const {viewMatrixUncentered, viewMatrix, projectionMatrix} = viewport;
 
   let projectionCenter;
-  let modelViewProjectionMatrix;
-  let modelViewMatrixInv;
-
-  const modelViewMatrix = new Matrix4(viewMatrix);
-  const viewProjectionMatrix = new Matrix4(projectionMatrix).multiplyRight(viewMatrix);
-
-  if (modelMatrix) {
-    modelViewMatrix.multiplyRight(modelMatrix);
-  }
+  let modelViewMatrix;
 
   switch (projectionMode) {
 
   case COORDINATE_SYSTEM.LNGLAT:
     projectionCenter = ZERO_VECTOR;
-    modelViewProjectionMatrix = viewProjectionMatrix;
-    if (modelMatrix) {
-      // Apply model matrix if supplied
-      // modelViewProjectionMatrix = modelViewProjectionMatrix.clone();
-      modelViewProjectionMatrix.multiplyRight(modelMatrix);
-    }
-    modelViewMatrixInv = new Matrix4(modelViewMatrix);
-    modelViewMatrixInv.invert();
+    modelViewMatrix = new Matrix4(viewMatrix);
```

Comment:
This now looks correct to me, although I think this code would be clearer if we didn't add the prefix `model` to the variableName until we actually multiply it in (which is now happening half a page down in the code). 

At this stage we should just call the variable `viewMatrix`.